nais/dev/dev.yaml

```diff
@@ -48,5 +48,7 @@ spec:
       value: "dev-gcp"
   accessPolicy:
     outbound:
+      external:
+        - host: dekoratoren.intern.dev.nav.no
```

Comment:
Hmm funka dette uten accesspolicy før?

Reply:
jeg troor ikke det slutter å funke før man deployer på nytt, så kanskje den ikke har vært deploya?

Reply:
for ser jo at prod ikke har vært ute også :O

Reply:
Forvirrende!